tests/workload/mountpoint.robot

```diff
@@ -44,11 +49,46 @@ Create a Mountpoint to a Nonexistent Volume
     [Teardown]    Run Keyword
     ...    Command Should Succeed        ${FS} rmm ${MTPT}
 
+Set RW ACL in root.cell for system:authuser
```

Comment:
Setting the ACL seems like it could belong in the Setup

Reply:
Will move that to the Setup